### PR TITLE
readme: list all our users

### PR DESCRIPTION
**Pull request by ligurio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -379,3 +379,10 @@ The following files will be removed:
 * `[0-9]*/`
 
 For a `core = unittest` test suite this option does not change any behaviour.
+
+### Used By
+
+- [Tarantool](https://github.com/tarantool/tarantool) - is an in-memory database and application server
+- [tarantool-python](https://github.com/tarantool/tarantool-python) - Python driver for Tarantool
```
Comment (Totktonada) [inline README.md:386]: It does not anymore: https://github.com/tarantool/tarantool-python/pull/189

That was just one test case, so don't count it.

File: README.md [hunk 2]
```diff
@@ -379,3 +379,10 @@ The following files will be removed:
 * `[0-9]*/`
 
 For a `core = unittest` test suite this option does not change any behaviour.
+
+### Used By
+
+- [Tarantool](https://github.com/tarantool/tarantool) - is an in-memory database and application server
```
Comment (Totktonada) [inline README.md:385]: Nit: I would remove 'is an' for consistency with the vshard description below.

File: README.md (hunk 1 above)
Comment (Totktonada) [reply README.md]: I applied the change myself to don't bother you. Hope you don't mind.

File: README.md (hunk 2 above)
Comment (Totktonada) [reply README.md]: I removed 'is an' here.

**Review by Totktonada — APPROVED** [review]
LGTM.

I made several minor changed to don't bother you with extra context switch. If you're okay, I'll push the change.

**Comment by ligurio** [comment]
Sure, I'm ok. Thanks!

**Pull request merged**